__tests__/user/user.mutation.test.js

```diff
@@ -766,3 +786,322 @@ describe('User`s mutation restictions tests', () => {
     expect(result.graphQLErrors[0].message).toEqual('USER_NOT_AUTHORIZED');
   });
 });
+
+describe("Creating users with custom roles",() => {
+    let superAdminToken;
+    let role;
+    let id;
+    let token;
+    let password;
+    let email;
+    let firstName;
+    let invalidFirstName;
+    let lastName;
+    let invalidEmail;
+    let adminEmail;
+    let invalidRole;
+    let invalidToken;
+
+    beforeAll(async () => {
+        superAdminToken = await adminLogin(superAdminUser);
+        role = 'admin';
```

Comment:
You can initialize these variables at the same place where you create it because they don't depend on the value of `adminLogin` call. Leave in `beforeAll` initializations only for the vars which values depend on the `adminLogin` call.

Reply:
fixed